### PR TITLE
Fix for #215. Allow delegate_attribute for derived classes

### PR DESCRIPTION
**Pull request by sbatykov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * [#215](https://github.com/ruby-grape/grape-entity/pull/217): Fix: `#delegate_attribute` no longer delegates to methods included with `Kernel` - [@maltoe](https://github.com/maltoe).
 * [#219](https://github.com/ruby-grape/grape-entity/pull/219): Fix: double pass options in serializable_hash - [@sbatykov](https://github.com/sbatykov).
 * [#226](https://github.com/ruby-grape/grape-entity/pull/226): Add fetch method to fetch from opts_hash - [@alanjcfs](https://github.com/alanjcfs).
+* [#231](https://github.com/ruby-grape/grape-entity/pull/231): Fix for #215 allow delegate_attribute for derived enyity - [@sbatykov](https://github.com/sbatykov)
```
Comment (dblock) [inline CHANGELOG.md]: If you're going to mention #215, link it the same way as the PR, but generally we just don't.

Entity is misspelled.

There needs to be a period at the end of this, see other lines;)

File: spec/grape_entity/entity_spec.rb [hunk 2]
```diff
@@ -1538,26 +1539,33 @@ def name
...
-        another_friend = double('Friend', email: 'joe@example.com')
-        rep = EntitySpec::DelegatingEntity.new(another_friend)
-        expect(rep.value_for(:name)).to eq 'cooler name'
-      end
+          another_friend = double('Friend', email: 'joe@example.com')
+          rep = EntitySpec::DelegatingEntity.new(another_friend)
+          expect(rep.value_for(:name)).to eq 'cooler name'
+        end
+
+        it 'Kernel methods does not delegate' do
```
Comment (dblock) [inline spec/grape_entity/entity_spec.rb]: Maybe fix the Englis here, "does not delegate Kernel methods"

**Comment by dblock** [comment]
This is great, just amend the text changes and I'll merge. Don't hesitate to squash the commits too. Thanks.

**Comment by anakinj** [comment]
It would be great to handle the delegation properly in the next version. Do you @sbatykov have the intention to finalize this?

**Comment by anakinj** [comment]
What is the status of this one, we are forced to do some ugly stuff to handle the degraded inheritance support #217 introduced.

**Comment by dblock** [comment]
Merged via https://github.com/ruby-grape/grape-entity/commit/22033a647215a33ec5865b40432e62d48fa54ab7. 

@anakinj Can you please check HEAD out before we cut a release?

**Comment by anakinj** [comment]
@dblock, works like a charm. Thanks for this.

**Pull request closed without merging**